lib/opensusebasetest.pm

```diff
@@ -1135,6 +1145,7 @@ sub post_fail_hook {
     return if testapi::is_serial_terminal();    # unless VIRTIO_CONSOLE=0 nothing below make sense
 
     show_tasks_in_blocked_state;
+    return if (get_var('NOLOGS'));
```

Comment:
Unrelated but, I have the feeling that this is a hack to `_SKIP_POST_FAIL_HOOKS`